appdatabase/migrations/sql/1671451982_add_balance_history.up.sql

```diff
@@ -0,0 +1,15 @@
+-- One entry per chain_id, address, currency, block
+-- bitset is used to select time interval granularity. The least significant bit has the coarsest granularity
+-- balance is a serialized big.Int
+CREATE TABLE IF NOT EXISTS balance_history (
+    chain_id UNSIGNED BIGINT NOT NULL,
+    address VARCHAR NOT NULL,
+    currency VARCHAR NOT NULL,
+    block BIGINT NOT NULL,
+    timestamp INT NOT NULL,
+    bitset INT NOT NULL,
+    balance BLOB
+);
```

Comment:
Good question, thanks for the heads up. My understanding is that the short currency is standardized and enough. The `token_address` for the contract is retrieved from our token store. Now I wonder if this can change. Can it be that the contract address for an ERC20 token can change without changing the short currency? Will ask around and look into.